PlanningPoker/Services/UserService.cs

```diff
@@ -1,159 +1,121 @@
-using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Security.Cryptography.X509Certificates;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.SignalR;
-using Microsoft.EntityFrameworkCore.Metadata.Internal;
 using PlanningPoker.Models;
-using Serilog;
+using PlanningPoker.Repostitories;
 
 namespace PlanningPoker.Services
 {
     public class UserService : IUserService
     {
-        private readonly List<Room> _rooms;
-        private readonly List<UserRoom> userRooms;
-        private List<UserVote> _usersVotes;
-        private List<UserConnection> _userConnections;
         private readonly IHubContext<LoopyHub> _hubContext;
+        private readonly IRoomsRepository _roomsRepository;
+        private readonly IUserRepository _userRepository;
+        private readonly IUnitOfWork _unitOfWork;
 
-        public UserService(IHubContext<LoopyHub> hubContext)
+        public UserService(IHubContext<LoopyHub> hubContext, IUnitOfWork unitOfWork, IRoomsRepository repository, IUserRepository userRepository)
         {
-            _rooms = new List<Room>();
-            _usersVotes = new List<UserVote>();
-            _userConnections = new List<UserConnection>();
-            userRooms = new List<UserRoom>();
             _hubContext = hubContext;
+            _roomsRepository = repository;
+            _userRepository = userRepository;
+            _unitOfWork = unitOfWork;
         }
 
-        public void AddVote(UserVote userVote)
+        public async void AddVote(string name, int vote)
```

Comment:
async void should be used only for event handlers. Please change void to Task and rename method to AddVoteAsync()